targets/esp8266/user/jerry_run.c

```diff
@@ -37,7 +37,7 @@ int js_entry (const char *source_p, const size_t source_size)
 
   js_register_functions ();
 
-  parsed_res = jerry_parse ((jerry_char_t *) jerry_src, source_size, false);
+  parsed_res = jerry_eval ((jerry_char_t *) jerry_src, source_size, false);
```

Comment:
Update error message 4 lines below. Or just drop the whole parse/eval functionality from this function and use this for jerry initialization only (and rewrite user_main.c to use `js_eval()` to run all scripts in `js_codes[]`).

And regarding initialization: don't forget to call `srand` as pointed out in #2056.

Reply:
I've added these modifies.

Reply:
I've found a bug that gettimeofday() always return with zero. I'm on to fix it.